code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1292,16 +1292,28 @@
 	damage_var_low = CONFIG_GET(number/combat_define/low_proj_variance)
 	damage_var_high = CONFIG_GET(number/combat_define/mlow_proj_variance)
 
+
 /datum/ammo/xeno/toxin/on_hit_mob(mob/living/carbon/M, obj/item/projectile/P)
+
 	if(!istype(M))
 		return ..()
 	var/mob/living/carbon/C = M
-	if(C.status_flags & XENO_HOST && istype(C.buckled, /obj/structure/bed/nest) || C.stat == DEAD)
+
+	if(C.status_flags & XENO_HOST && istype(C.buckled, /obj/structure/bed/nest) || C.stat == DEAD || !(ishuman(C) || ismonkey(C)))
```

Comment:
I'd suggest you move the `(ishuman(C) || ismonkey(C))` check just above the ammo reagents magic instead, because this would make #755 useless as of now.